.github/workflows/compilation.yml

```diff
@@ -19,120 +19,157 @@ jobs:
 
     - name: Compile -> make RTL=1 PADEMU=1 clean OPNPS2LD.ELF
       run: |
         make RTL=1 PADEMU=1 clean OPNPS2LD.ELF
-        mv OPNPS2LD.ELF OPNPS2LD-RTL-PADEMU.ELF
+        mv OPNPS2LD.ELF OPNPS2LD-RTL-PADEMU-${{ steps.version.outputs.version }}.ELF
 
     - name: Compile -> make RTL=1 PADEMU=1 DTL_T10000=1 clean OPNPS2LD.ELF
       run: |
         make RTL=1 PADEMU=1 DTL_T10000=1 clean OPNPS2LD.ELF
-        mv OPNPS2LD.ELF OPNPS2LD-RTL-PADEMU-DTL_T10000.ELF
+        mv OPNPS2LD.ELF OPNPS2LD-RTL-PADEMU-DTL_T10000-${{ steps.version.outputs.version }}.ELF
 
     - name: Compile -> make RTL=1 IGS=1 clean OPNPS2LD.ELF
       run: |
         make RTL=1 IGS=1 clean OPNPS2LD.ELF
-        mv OPNPS2LD.ELF OPNPS2LD-RTL-IGS.ELF
+        mv OPNPS2LD.ELF OPNPS2LD-RTL-IGS-${{ steps.version.outputs.version }}.ELF
 
     - name: Compile -> make RTL=1 IGS=1 DTL_T10000=1 clean OPNPS2LD.ELF
       run: |
         make RTL=1 IGS=1 DTL_T10000=1 clean OPNPS2LD.ELF
-        mv OPNPS2LD.ELF OPNPS2LD-RTL-IGS-DTL_T10000.ELF
+        mv OPNPS2LD.ELF OPNPS2LD-RTL-IGS-DTL_T10000-${{ steps.version.outputs.version }}.ELF
 
     - name: Compile -> make RTL=1 IGS=1 PADEMU=1 clean OPNPS2LD.ELF
       run: |
         make RTL=1 IGS=1 PADEMU=1 clean OPNPS2LD.ELF
-        mv OPNPS2LD.ELF OPNPS2LD-RTL-IGS-PADEMU.ELF
+        mv OPNPS2LD.ELF OPNPS2LD-RTL-IGS-PADEMU-${{ steps.version.outputs.version }}.ELF
 
     - name: Compile -> make RTL=1 IGS=1 PADEMU=1 DTL_T10000=1 clean OPNPS2LD.ELF
       run: |
         make RTL=1 IGS=1 PADEMU=1 DTL_T10000=1 clean OPNPS2LD.ELF
-        mv OPNPS2LD.ELF OPNPS2LD-RTL-IGS-PADEMU-DTL_T10000.ELF
+        mv OPNPS2LD.ELF OPNPS2LD-RTL-IGS-PADEMU-DTL_T10000-${{ steps.version.outputs.version }}.ELF
+
+    - name: Create detailed changelog
+      run: sh ./make_changelog.sh
+
+    - name: Create zip archive
+      run: zip -r OPNPS2LD-VARIANTS-LATEST.ZIP OPNPS2LD*.ELF DETAILED_CHANGELOG CREDITS LICENSE README.md
 
     - name: Upload variants artifact
       if: ${{ success() }}
       uses: actions/upload-artifact@v2
       with:
-        name: OPNPS2LD-VARIANTS-${{ steps.slug.outputs.sha8 }}
-        path: OPNPS2LD*.ELF
+        name: OPNPS2LD-VARIANTS
+        path: |
+          OPNPS2LD-VARIANTS-LATEST.ZIP
+
+  create-release:
+    needs: [build, build-variants]
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        git fetch --prune --unshallow
+
+    - name: Download release artifact
+      uses: actions/download-artifact@v1
+      with:
+        name: OPNPS2LD
+
+    - name: Download variants artifact
+      uses: actions/download-artifact@v1
+      with:
+        name: OPNPS2LD-VARIANTS
+
+    - name: Create release
+      uses: marvinpinto/action-automatic-releases@latest
+      with:
+        repo_token: "${{ secrets.GITHUB_TOKEN }}"
+        automatic_release_tag: "latest"
+        title: "Latest development builds"
```

Comment:
As I see there is no `body` support int this Github Action. Maybe in the future, it will be useful to switch to another action with `body` support, so more information about the release can be provided (such as compilation date, latest commit info, checksums, dependency's info, etc).

Reply:
If there is such an action, let me know.